backend/codesirius/problems/views/problem.py

```
@@ -0,0 +1,132 @@
+import logging
+
+from rest_framework import status
+from rest_framework.exceptions import ValidationError, NotFound, APIException
```

Ruff (0.8.2)

`rest_framework.exceptions.APIException` imported but unused

Remove unused import: `rest_framework.exceptions.APIException`

(F401)